test/integration/puppetlabs/general_puppet/general_puppet_int_test.clj

```diff
@@ -20,6 +20,12 @@
 
 (def num-jrubies 1)
 
+(defn get-static-file-content
+  [url-end]
+  (http-client/get (str "https://localhost:8140/puppet/v3/static_file_content/" url-end)
+                   (assoc testutils/ssl-request-options
+                     :as :stream)))
```

Comment:
This can be replaced with `:as :text` which would remove the need to `slurp` the response body everywhere it's used.